src/components/search_bar/filters/field_value_selection_filter.tsx

```diff
@@ -37,7 +37,7 @@ import { OuiFilterButton, OuiFilterSelectItem } from '../../filter_group';
 import { OuiLoadingChart } from '../../loading';
 import { OuiSpacer } from '../../spacer';
 import { OuiIcon } from '../../icon';
-import { Query } from '../query';
+import { Query } from '../query/query';
```

Comment:
Do we already export `Query` for `../query`? If not, it seems like that might be desirable. Even more so for backwards compatibility reasons?

Reply:
We have `index.ts`, but it doesn't work for class imports and that's weird

https://github.com/opensearch-project/oui/blob/ce8acf41cc58cccbef180d1d93e0025e29e4a312/src/components/search_bar/query/index.ts#L31-L32

```bash
WARNING in ../../src/components/search_bar/search_bar.tsx 256:39-44
export 'Query' (imported as 'Query') was not found in './query' (possible exports: AST)
 @ ../../src/components/search_bar/index.ts 31:0-49 31:0-49 31:0-49
 @ ../../src/components/index.js 94:0-49 94:0-49 94:0-49
 @ ./routes.js 56:0-56 205:44-60
 @ ./index.js 42:0-30 58:28-47
```